assets/components/paymentButtons/stripePopUpButton/stripePopUpButton.scss

```diff
@@ -25,8 +23,15 @@
     top: 50%;
     transform: translateY(-50%);
   }
+  .svg-credit-card {
+    fill: gu-colour(garnett-neutral-1);
+  }
+
+  @include mq($from: phablet) {
+    width: 304px;
+  }
```

Comment:
I think this depends on the layout of the page where the button is used. For example, it's this value at the moment to match the widths of the entry fields and the PayPal button, but if it were to be used somewhere else it may need to be a different width to match other components on that page.